uaclient/event_logger.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
```

Comment:
Do we need this line?

Reply:
To be honest, I don't think we need it. I just followed the "template" of the `cli` module. But I agree we can remove it